nautobot/virtualization/forms.py

```diff
@@ -528,6 +543,20 @@ class VMInterfaceCreateForm(BootstrapMixin, InterfaceCommonForm):
     virtual_machine = DynamicModelChoiceField(queryset=VirtualMachine.objects.all())
     name_pattern = ExpandableNameField(label="Name")
     enabled = forms.BooleanField(required=False, initial=True)
+    parent_interface = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(),
+        required=False,
+        query_params={
+            "virtual_machine_id": "$virtual_machine",
+        },
+    )
+    bridge = DynamicModelChoiceField(
+        queryset=VMInterface.objects.all(),
+        required=False,
+        query_params={
+            "virtual_machine_id": "$virtual_machine",
+        },
+    )
```

Comment:
Please add `help_text` here too. Altho there's not there for any others, we should assert that new fields have it.